### ISSUE TITLE
"This game is present in your F95Checker library!" icon not showing for existing entries 

### ISSUE THREAD

**Issue by FaceCrap** [Willy-JL/F95Checker#46]
F95Checker Build 861
installed the new extension per ReadMe.md 
Browsers tried: Opera GX, Google Chrome

After added two games to F95Checker, I noticed that one thread showed the new icon, but the second thread didn't and also didn't show after reloading/reopening
Seeing this new icon, I immediately went and opened some threads from already existing games within F95Checker and noticed none of these showed that icon either.
It will only show for existing entries in F95Checker if I click on the extension icon, consequently getting the error that there already is a duplicate game present, after which the icon then shows up on the thread title, but only as long as the thread is still opened. If I close the thread and re-open it from within F95Checker, the icon is gone again.
Apart from that, it seems that when adding a new game through the extension, the icon also doesn't always show. If this happens, the above steps (clicking the extension icon again) will then fix it, but again, only for as long as the thread stays open. 

In short, every time I open an existing entry from within F95Checker, the icon is missing.

**Comment by Willy-JL** [comment]
Don't know what to say, I can't manage to reproduce this.

I tried opening about 100 games from my list in batch and never had this happen.

Or better said, I had this happen temporarely: the icons are added on page load finish, so if the tab shill shows as loading it wont try to add the icon. But for me as soon as each single tab finished loading the icon was there

**Comment by FaceCrap** [comment]
Whatever you changed between the latest release and build 861... 

1. Adding the extension again to Opera GX no longer throws the error and the permission is now listed in the details 
2. Every thread I launched from F95Checker does indeed now have the icon in front of the thread title

so this one was probably because of the weird permission error
